packages/@react-spectrum/color/src/ColorArea.tsx

```diff
@@ -59,13 +58,12 @@ function ColorArea(props: SpectrumColorAreaProps, ref: FocusableRef<HTMLDivEleme
       }
       ref={containerRef}
       style={{
-        ...colorAreaProps.style,
+        ...(isDisabled ? {} : colorAreaProps.style),
```

Comment:
Previously no gradient was returned by the hook when disabled, but I don't think we can assume that all color areas will work that way. This moves that logic into Spectrum instead.